chart/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Do we need this? There's a way of pre-populating the secret see -> https://github.com/petbattle/tournamentservice-v1/wiki/Tournament-Service-Install-&-Config
Yea I know semantics but I don't like checking in secrets if possible

Reply:
Yeah, this was a dirty workaround for now ... a wiki .. i didn't even know that existed ;) Yeah .. ultimately we need secrets - your identities.yaml is the same thing ... lolz.

ultimately the Chart could be doing too many things - its installing the applications dependencies (mongo, keycloak, infinispan) - these we could/should all pull out into separate deployments - then it may be cleaner to connect using wellknown co-ordinates and oc set env from secrets that the infra creates ? more work required ...